src/main/java/picard/sam/CleanSam.java

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2010 The Broad Institute
+ * Copyright (c) 2010 - 2016 The Broad Institute
```

Comment:
What's so special about 2016 here?  Why not through 2021?

Reply:
goes to show when the code was written...I guess.